docs/DeveloperGuide.adoc

```diff
@@ -50,7 +50,11 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |add a picture to a person |better able to recognize who the contact person is
+|`* * *` |user |add labels to person |categorize the contacts
```

Comment:
Perhaps this could use a more specific role, such as as a `user who likes to be organized`...